src/PrestaShopBundle/Form/Admin/Catalog/Category/AbstractCategoryType.php

```diff
@@ -108,10 +133,9 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                     ],
                 ],
             ])
-            ->add('description', TranslateType::class, [
+            ->add('description', TranslatableType::class, [
+                'label' => $this->trans('Description', 'Admin.Global'),
                 'type' => FormattedTextareaType::class,
```

Comment:
Note that I removed the help text "Invalid characters: <>;=#{}". 
That help text doesn't make sense because the only constraint here is CleanHtml which will check for script tags and such but will pass "<>;=#{}". 

I don't know if new help text is needed to reflect that.

Reply:
You are right, if these characters are allowed, we should get rid of the help text. I don't think it is necessary to have a help text at all in that case. ☺️